src/components/chip/chip.classnames.ts

```diff
@@ -0,0 +1,67 @@
+import { ChipSize } from './types'
+import { cva } from 'class-variance-authority'
+
+export const chipBaseVariant = cva('button', {
```

Comment:
I don't think `button` is a valid base classname correct if I'm wrong. I'm sure you meant to put instead
```suggestion
export const chipBaseVariant = cva('chipBaseClassnames', {
```

Reply:
You are right.

Reply:
done ✅